src/parameters/update_parameters.jl

```diff
@@ -166,7 +166,7 @@ function _update_parameter_values!(
     attributes::VariableValueAttributes,
     ::Type{<:PSY.Component},
     model::DecisionModel,
-    state::DatasetContainer{DataFrameDataset},
+    state::DatasetContainer{InMemoryDataset},
```

Comment:
This function needs to be tested or check a case where this is called. Potentially delete or comment out.